src/update_engine/simple_key_value_store.cc

```diff
@@ -27,7 +27,14 @@ map<std::string, std::string> ParseString(const string& str) {
     string::size_type pos = it->find('=');
     if (pos == string::npos)
       continue;
-    ret[it->substr(0, pos)] = it->substr(pos + 1);
+    string val = it->substr(pos + 1);
+    if ((val.length() > 2) &&
```

Comment:
I'd probably use `>=`, so `FOO=""` would end up storing empty value.
```suggestion
    if ((val.length() >= 2) &&
```